csuntechlab/iSTART
# STR-231 -- FE: Implement the Comparison Module

# Description
Implement comparison module.
  
# Testing
1. Make sure the new comparison module is implemented.
2. Ensure the content fits the templates.
